### ISSUE TITLE
Adding links of related articles in mdx files. 

### ISSUE THREAD

**Issue by Gitstar-OC** [Gitstar-OC/Mindect#13]
**This is a beginner friendly issue on which any one can work on**
> You can contribute either you know this topic or not in Machine Learning 

Here's what you are supposed to do:  
- At first choose any topic in `/content/docs` folder. 
 For example If you choose the gradient descent folder, the main file is index.mdx in it and you will also find some other things in the folder of gradient descent. Select a file in it. 
-  Let's say you choose `Learning Rate (α)` file. In the file check if [this issue](https://github.com/Gitstar-OC/mindect/issues/8) is solved which is also beginner friendly, if not you can choose to solve it or do the following tasks. 
- After choosing and reading the file, you have to search for the particular articles on the web about it. 

  In this case that would be 
![Screenshot 2024-09-26 121429](https://github.com/user-attachments/assets/aa74f0a3-a35c-44f5-b360-8936512a3b9b)

- Just open the top links (wikipedia recommended) and then just add a new heading in the learning rate markdown file as additional resources, extra links, read more about this etc. 
  That will look like this
  ![Screenshot 2024-09-26 122049](https://github.com/user-attachments/assets/df75cbe1-e94d-43f7-9143-0565dc86034d)

- Which will look like this in the development and production. 
![Screenshot 2024-09-26 122144](https://github.com/user-attachments/assets/898b091f-7674-4ff2-9efd-4575793da847)

**If you want to contribute you should do this for minimum 4 files of you choice** and note that **While implementing this addition no other code should be changed or removed** There should just be addition.
